indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -185,3 +188,31 @@ class RHRegistrationEdit(RHManageRegistrationBase):
     def _process(self):
         return WPManageRegistration.render_template('management/registration_modify.html', self.event,
                                                     registration=self.registration)
+
+
+class RHRegistrationEmailRegistrants(RHManageRegFormBase):
+
+    def _checkParams(self, params):
+        self._doNotSanitizeFields.append('from_address')
+        RHManageRegFormBase._checkParams(self, params)
+
+    def _get_people(self):
+        identifiers = set(request.form.getlist('references'))
+        return Registration.find_all(Registration.id.in_(identifiers))
```

Comment:
Yes, it won't work. find_all() is `find().all()` so it returns you a list. And you can't call query methods on a list...